docs/hydra/guides/openid.md

```diff
@@ -18,49 +19,43 @@ ory patch oauth2-config {project.id} \
 
 ## Pairwise algorithm
 
-The `pairwise` algorithm provides a different sub value to each client, ensuring that the clients cannot correlate the user's
-activities without permission. To use the `pairwise` algorithm, you must enable it and set
-`/oidc/subject_identifiers/pairwise/salt`. This salt is used to obfuscate the `sub` value.
+The `pairwise` algorithm provides a different subvalue to each client, ensuring that clients can't correlate user activity without
+permission. To use the `pairwise` algorithm, you must enable it and set `/oidc/subject_identifiers/pairwise/salt`. The salt is
+used to obfuscate the `sub` value.
 
-To enable both algorithms, use the following configuration:
+To enable the `pairwise` algorithm, run this command:
 
 ```shell
 ory patch oauth2-config {project.id} \
-  --replace "/oidc/subject_identifiers/supported_types=[\"public\", \"pairwise\"]" \
+  --replace "/oidc/subject_identifiers/supported_types=[\"pairwise\"]" \
   --replace "/oidc/subject_identifiers/pairwise/salt=\"{16-character-long-salt}\""
 ```
 
 :::warning
 
-Do not change the salt value once it is set in production. Changing it will cause all client applications to receive new user IDs
-from Ory, which can lead to serious complications with authentication.
+Don't change the salt value once it's set in production. When you change the salt, all client applications receive new user IDs
```

Comment:
Not really an important point, but in this context the uncontracted form of "do not" has a natural rhythm to it. The uncontracted form has more emphasis, which is natural for a Warning.

Reply:
I think this is mostly bc we follow the microsoft styleguide here: 
https://learn.microsoft.com/en-us/style-guide/word-choice/use-contractions

So basically always contract it. I do agree though that it gives a different rhythm when not contracted...

Reply:
What Vincent said is true. I'm using contractions on everything for consistency. Any benefits of not contracting in a case like this are negligible IMO.